metaci/hirefire/views.py

```diff
@@ -29,7 +29,7 @@ def info(request, token):
     if not settings.HIREFIRE_TOKEN:
         return HttpResponseBadRequest('Hirefire not configured.  Set the HIREFIRE_TOKEN environment variable on the app to use Hirefire for dyno scaling')
     if token != settings.HIREFIRE_TOKEN:
-        return HttpResponseForbidden('Invalid token')
```

Comment:
This one I think is actually fine, since it's going to end up in a log somewhere on hirefire, we don't need the full template